frameit/lib/frameit/editor.rb

```diff
@@ -233,27 +231,20 @@ def put_title_into_background_stacked(background, title, keyword)
       resize_text(title)
       resize_text(keyword)
 
-      title_width = title.width
-      keyword_width = keyword.width
-
-      vertical_padding = vertical_frame_padding
-      keyword_top_space = vertical_padding
```

Comment:
Didn't this line indicate that the vertical frame padding was _also_ used as the "keyword top space" (which probably means margin-top for the keyword I guess?)

Reply:
What I mean: Using the "vertical_frame_padding" in line 244 doesn't make too much sense, but adding a "keyword margin" or something does make sense. So having some code that makes sure that these two values are identical only looks useless in the first moment, but not when trying to understand what all this code means.

Reply:
Same for the `vertical_frame_padding` variable/method by the way: It is used to get a number, but the number is not used as `frame_padding` any more.

Reply:
(Could it be that you are calling this method over and over right now?)

Reply:
You're absolutely right. This proofs:
1. my lack of Ruby experience; I can't even discern a variable from a method.
2. there is no proper naming convention in this Class to create that discern.

I'm used to other languages (such as Java, Obj-C, C#) where there is a syntax difference between calling a variable and a method (often by parenthesis in method calls).
So it doesn't help that this Editor Class has a variety of methods with names that imply a property/variable, such as `vertical_frame_padding`. 

I'm taught that a method name always contains a verb. Such as "get_padding", "move_frame_into_container", etc.

Q: Are there any code guidelines and naming conventions for the Fastlane project (or Ruby in general)?

Indeed method `vertical_frame_padding` (and others) are often called in this Class. That's unnecessary and increases computation time, especially since it are often intense File I/O calls to fetch config data from the Framefile.

As this code review comment editor doesn't show the Advanced Editing menu, I'll create a new comment which does so....

Reply:
I have no idea, I am confused by the variable/method thing myself. A few commits ago

Reply:
I made some measurements, each call to method `vertical_frame_padding` requires 6 to 15 microseconds (e.g. 0.000011 s).
Its called 5 times per screenshot. So the computation time isn't really significant.

What do you want for now?
Do you want me to restore the local params?
I prefer to leave this PR as is, as it fixes issue #11585 which is confirmed by @JosephDuffy.
I will open a new issue to improve the code, which we can do in cooperation with some Ruby experts from the Team.

Reply:
NB: Measurements were done on a MacBook Pro (Retina, 13-inch, Early 2015) with 2,7 GHz Intel Core i5 and 16 GB RAM.